train.py

```diff
@@ -51,12 +53,14 @@
     "etci2021": (SemanticSegmentationTask, ETCI2021DataModule),
     "inria": (SemanticSegmentationTask, InriaAerialImageLabelingDataModule),
     "landcoverai": (SemanticSegmentationTask, LandCoverAIDataModule),
+    "loveda": (SemanticSegmentationTask, LoveDADataModule),
```

Comment:
I don't believe SemanticSegmentationTask supports LoveDADataModule. I think I tried adding it to our tests but it doesn't work.

Reply:
I trained models with it and it worked IRL -- I think this is probably a test issue

Reply:
Hmm, can you remove this change from this PR and add it to a separate PR where you also try to get the tests working? I'm hesitant to put things in `train.py` if we're unable to test them properly, and it seems unrelated to this PR anyway.

Reply:
sure